vcd/datasource_vcd_vapp_vm.go

```diff
@@ -375,6 +375,12 @@ func vcdVmDS(vmType typeOfVm) map[string]*schema.Schema {
 			Computed:    true,
 			Description: "VM sizing policy ID.",
 		},
+		"placement_policy_id": {
+			Type:        schema.TypeString,
+			Optional:    true,
```

Comment:
I'm reading the PR description and from "Use cases" section it seems as if you can't create a VM without neither the sizing_policy_id nor placement_policy_id. Maybe worth stating that option in PR description?

Reply:
Done!